### PR TITLE
[Bug] In login page, tab from email input does not go to input password

### PR DESCRIPTION
**Pull request by maurohmartinez**
Everything [here](https://github.com/orgs/Laravel-Backpack/projects/13/views/10).

### CODE REVIEW

File: resources/views/auth/login/cover.blade.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
                 @include(backpack_view('auth.login.inc.form'))
                 @if (config('backpack.base.registration_open'))
                     <div class="text-center text-muted mt-3">
-                        <a href="{{ route('backpack.auth.register') }}" tabindex="-1">{{ trans('backpack::base.register') }}</a>
+                        <a tabindex="6" href="{{ route('backpack.auth.register') }}" tabindex="-1">{{ trans('backpack::base.register') }}</a>
```
Comment (pxpm) [inline resources/views/auth/login/cover.blade.php:13]: This has now two tabindex attributes.

Comment (maurohmartinez) [reply resources/views/auth/login/cover.blade.php]: Ops... good catch!

File: resources/views/auth/login/cover.blade.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
                 @include(backpack_view('auth.login.inc.form'))
                 @if (config('backpack.base.registration_open'))
                     <div class="text-center text-muted mt-3">
-                        <a href="{{ route('backpack.auth.register') }}" tabindex="-1">{{ trans('backpack::base.register') }}</a>
+                        <a tabindex="6" href="{{ route('backpack.auth.register') }}">{{ trans('backpack::base.register') }}</a>
                     </div>
                 @endif
```
Comment (pxpm) [inline resources/views/auth/login/cover.blade.php:15]: This bit of code is the same on all 3 layouts (cover, default and illustration). 

Wouldn't be better to make it a partial and include it ? `@includeWhen(config('backpack.base.registration_open'))`

**Comment by tabacitu** [comment]
Heads-up @maurohmartinez , the project board is a moving thing, it always changes. By the time someone clicks the link, the issues you fixed here won't be there any more, so it doesn't make sense to link to it. 

I think a screenshot of the board would have worked better here 😉 Or individual links to issues.

File: resources/views/auth/login/cover.blade.php (hunk 2 above)
Comment (tabacitu) [reply resources/views/auth/login/cover.blade.php:15]: Neeah... too small to split out

**Comment by tabacitu** [comment]
I see Mauro is still assigned, but I'm going to assume this is ready so I'm going to merge.

**Pull request merged**